doc/tutorials/building_acrn_in_docker.rst

```diff
@@ -0,0 +1,158 @@
+.. _building-acrn-in-docker:
+
+Building ACRN in Docker
+#######################
+
+In this tutorial, we will introduce how to build ACRN in clearlinux docker image.
+
+Install Docker
+**************
+
+#. Install docker according to `docs to docker <https://docs.docker.com/install/>`_.
+#. If you are behind HTTP or HTTPS proxy server, accroding to
+   `HTTP/HTTPS proxy <https://docs.docker.com/config/daemon/systemd/#httphttps-proxy>`_
+   to setup HTTP/HTTPS proxy to pull docker image and
+   `Configure Docker to use a proxy server <https://docs.docker.com/network/proxy/>`_
+   to setup HTTP/HTTPS proxy for docker container.
```

Comment:
``setup`` is a noun, change to ``set up``,  also change ``docker`` to ``the Docker``